Vagrantfile

```diff
@@ -122,7 +117,7 @@ Vagrant.configure("2") do |config|
       ansible.verbose = 'v'
       # the production playbook verifies that staging default values are not
       # used will need to skip the this role to run in Vagrant
-      ansible.skip_tags = [ "validate" ]
+      ansible.skip_tags = ENV['PROD_SKIP_TAGS']
```

Comment:
Intentionally absent so that future prod tests would have to skip the validate role explicitly. I'm no longer convinced that's practical, however. Once the prod tests are finalized, I'll address whether a failover is appropriate, but I don't think it needs to be into this PR.